blobfuse/utilities.cpp

```diff
@@ -259,10 +234,7 @@ std::vector<list_blobs_hierarchical_item> list_all_blobs_hierarchical(std::strin
         {
             failcount++;
             success = false;
-            if (AZS_PRINT)
-            {
-                fprintf(stdout, "list_blobs_hierarchical failed %d time with errno = %d\n", failcount, errno);
-            }
+            syslog(LOG_WARNING, "list_blobs_hierarchical failed for the %d time with errno = %d.\n", failcount, errno);
```

Comment:
warning only when it fails ?

Reply:
Yes, because it might succeed on retry... this stuff should be ripped out anyway, once cpplite has better retry support.

Reply:
Ok - I think we have a syslog statement for failed readdirs.. So it is fine